watchman/checks.py

```diff
@@ -81,3 +95,7 @@ def email():
 
 def storage():
     return {"storage": _check_storage()}
+
+
+def amqp():
+    return {'amqp': _check_ampq(settings.AMQP_CONNECTIONS)}
```

Comment:
The main thing that's an issue to me is.. how many users of this package would actually be able to use this check and get any value out of it? My guess is very low - which pushes me towards a separate package for this.